### DISCUSSION TITLE
New Game Ideas

### DISCUSSION THREAD

**Discussion by xavier506** [eoscostarica/gGoods#195] [Ideas]
Lets brainstorm on new ideas for games built on the ggoods platform

**Comment by xavier506** [comment]
Gdog ! its a new NFT template that takes sound clips , they are dogs barks that can be tokenized and sold to support the animal shelter.

**Comment by kriskoin** [comment]
the same app can collect funds for the animal necessities. ex: need to be relocated to its home

**Comment by kriskoin** [comment]
an app to learn how to "talk | sing " with birds|gods |cows|cats
